src/eradiate/scenes/phase/_tabulated.py

```diff
@@ -134,15 +164,15 @@ def eval_mono(self, w: pint.Quantity) -> np.ndarray:
         w_units = self.data.w.attrs["units"]
 
         return (
-            self.data.isel(i=0, j=0)
+            self.data.isel(i=i, j=j)
             .interp(
                 w=w.m_as(w_units),
                 kwargs=dict(bounds_error=True),
             )
             .data
         )
 
-    def eval_ckd(self, w: pint.Quantity, g: float) -> np.ndarray:
+    def eval_ckd(self, w: pint.Quantity, i: int, j: int, g: float) -> np.ndarray:
```

Comment:
I'd prefer to append the new dimensions at the end of the list if possible:
```suggestion
    def eval_ckd(self, w: pint.Quantity, g: float, i: int, j: int) -> np.ndarray:
```
This way, we keep spectral coordinates clustered in the signature.